src/machine/machine_rp2040.go

```diff
@@ -76,6 +76,8 @@ func machineInit() {
 
 	clocks.init()
 
+	RTC.init()
```

Comment:
Best not to initialize unconditionally like this - in general, enable peripherals only if the app will use them

Reply:
OK, will add a check and init in `SetTime()` then.